src/main/java/frc/lib/util/AprilTag.java

```diff
@@ -59,8 +59,8 @@ public void aprilTagPeriodic() {
   // Fetches, calculates, and returns the offset from the robot to the current primary apriltag target.
 
   public Optional<Transform3d> targetPos() {
-        // Optional<double[]> targetPose = Optional.of(entry("targetpose_robotspace").get().getDoubleArray(new double[0]));
-        Optional<double[]> targetPose = Optional.of(new double[] {-5, 0, 0, 0, 0, 0});
+        Optional<double[]> targetPose = Optional.of(entry("targetpose_robotspace").get().getDoubleArray(new double[0]));
```

Comment:
will not work if #getDoubleArray returns an array of zeroes (which it does often). should probably filter the optional (you can also use Optional#map here since you turn an optional into another optional).